packages/ref-core/tests/unit/test_datasets.py

```diff
@@ -11,6 +11,14 @@ def dataset_collection(cmip6_data_catalog) -> DatasetCollection:
     )
 
 
+@pytest.fixture
+def dataset_collection2(obs4mips_data_catalog) -> DatasetCollection:
```

Comment:
```suggestion
def dataset_collection_obs4mips(obs4mips_data_catalog) -> DatasetCollection:
```

This probably needs changes elsewhere, but is more explicity

Reply:
Newest commit should fix this.